content/components/spinner.mdx

```diff
@@ -46,12 +46,37 @@ The size of the spinner can be adjusted. The **medium** (default) variant works
 
 </Box>
 
-## Accessibility
+## Accessibility and usability expectations
 
-By default, our spinner component provides a generic loading status announcement for assistive technologies. We recommend that you specify a custom announcement specific to your use-case. For example: "Loading changed files".
+Use the `Spinner` component to provide a visible loading indicator for situations where an asynchronous API call or process may take a moderate amount of time (between 1 and 3 seconds).
```

Comment:
> between 1 and 3 seconds

What guidance do we have for people who anticipate a request will be under 1s or over 3s?

Reply:
[https://primer.style/ui-patterns/loading#determinate-loading-indicators left that up to the reader](https://primer.style/ui-patterns/loading#adapting-to-different-wait-times)

It might be worth adding a reference to this here?

Reply:
```suggestion
Use the `Spinner` component to provide a visible loading indicator for situations where an asynchronous API call or process may take a moderate amount of time (between 1 and 3 seconds) – see [Adapting to different wait times](https://primer.style/ui-patterns/loading#adapting-to-different-wait-times)](https://primer.style/ui-patterns/loading#adapting-to-different-wait-times).
```